tests/test_mechanics.py

```diff
@@ -279,6 +279,31 @@ def test_discard_enchanted_cards():
 	assert not game.player1.hand
 
 
+def test_discover():
```

Comment:
Since we're implementing neutral hero functionality, can we also test that it has the right behavior?